.travis.yml

```diff
@@ -1,14 +1,17 @@
 language: cpp
 compiler: gcc
-cache: ccache
+cache:
+  ccache: true
+  directories:
+    - /var/cache/apt/archives
 sudo: required
 dist: trusty
 install:
-  - ./utils/travis-dependencies.sh
+  - ./utils/travis/dependencies.sh
 script:
-  - cd utils
-  - source /opt/qt*/bin/qt*-env.sh
-  - ./travis-build.sh
+  - . utils/travis/common.sh
+  - env_setup
+  - ./utils/travis/build.sh
```

Comment:
Can you maybe also only run travis/build.sh if a cmake or qmake build is done? If we are already restrructuring everything ... 
I saw that you can run individual build commands for each matrix element in the minetest repository.

Reply:
Of course you can, and if you set one, it'll overwrite the default configuration.

Reply:
:+1: